packages/chat-core-aws-connect/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "@yext/chat-core-aws-connect",
+    "version": "0.1.0",
+    "description": "Typescript Networking Library for the Yext Chat API Integration with Amazon Connect",
+    "main": "./dist/commonjs/index.js",
+    "module": "./dist/esm/index.mjs",
+    "types": "./dist/esm/index.d.ts",
+    "sideEffects": false,
+    "keywords": [
+        "networking",
+        "chat api",
+        "chat",
+        "browser",
+        "nodejs",
+        "typescript",
+        "commonjs",
+        "es6",
+        "yext"
+    ],
+    "files": [
+        "dist",
+        "src",
+        "THIRD-PARTY-NOTICES",
+        "LICENSE"
+    ],
+    "scripts": {
+        "test": "jest --config=jest.config.json",
+        "lint": "prettier --write . && eslint --fix --max-warnings=0 .",
+        "tsc": "tsc -p tsconfig.json",
+        "dev": "npm run tsc -- --watch",
+        "generate-notices": "../../generate-notices.sh",
+        "generate-docs": "api-extractor run --local --verbose && api-documenter markdown --input-folder temp --output-folder docs && rm -rf temp",
+        "build:js": "rollup --config ../../rollup.config.mjs",
+        "build": "rm -rf dist/** && npm run build:js && npm run generate-docs && npm run generate-notices"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/yext/chat-core.git"
+    },
+    "author": "clippy@yext.com",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://github.com/yext/chat-core/issues"
+    },
+    "homepage": "https://github.com/yext/chat-core#readme",
+    "dependencies": {
+        "amazon-connect-chatjs": "^2.3.0",
+        "cross-fetch": "^3.1.5",
+        "dotenv": "^16.4.5"
```

Comment:
I don't think dotenv is needed for the library bundle itself right? if it's only for the build process, can you move it to devDependencies